manifests/candlepin.pp

```diff
@@ -130,6 +121,30 @@
       key_mode    => '0440',
     }
 
+    file { $keystore_password_path:
+      ensure  => file,
+      content => $keystore_password,
```

Comment:
unrelated to this PR (as this is old code), but does this leak the password to the logs?

Reply:
Solid question -- https://github.com/theforeman/puppet-certs/pull/337

Reply:
Marking as unresolved. While that PR was merged, this now drops the `show_diff` again.